mongodb/src/main/java/com/yahoo/ycsb/db/MongoDbClient.java

```diff
@@ -272,7 +275,8 @@ public int insert(String table, String key,
                 BulkWriteResult result = collection.withWriteConcern(
                         writeConcern).bulkWrite(bulkInserts,
                         new BulkWriteOptions().ordered(false));
-                if (result.getInsertedCount() == bulkInserts.size()) {
+                if (!result.wasAcknowledged()
```

Comment:
Is this correct? It looks like you'll return 0 if the result was *not* acknowledged.